plugins/Jobs.cpp

```diff
@@ -345,6 +358,14 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
     STable& content = command.jsonContent;
     const string& requestVerb = request.getVerb();
 
+    if (!lastJobID) {
```

Comment:
I don't think this is the best place for this. Initially I wanted to add it to the `initialize` method, but I did not have access to the db object there... let me know if there's a better place to put this.

Reply:
What about doing it in `upgradeDatabase`?

Reply:
I thought about it, seems just as misplaced as here, but I can move it there if you think is better.

Reply:
At least there it only gets run once at startup instead of on every request.